File: tanya/src/main/java/hu/szakdolgozat/tanya/service/dto/TaskMiniDTO.java
package hu.szakdolgozat.tanya.service.dto;

import hu.szakdolgozat.tanya.entity.enumeration.IssueStatus;
import hu.szakdolgozat.tanya.entity.enumeration.IssueType;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class TaskMiniDTO {

	private Long id;

	private String issueName;

	private String createUserName;

	private String responsibleUserName;

	private IssueStatus issueStatus;

	private IssueType issueType;
}

File: tanya/tanya-frontend/src/app/model/task.mini.dto.ts
import { IssueType } from './enum/issue-type.enum';
import { IssueStatus } from './enum/issue-status.enum';

export interface TaskMiniDTO {
    id: number;
    issueName: string;
    createUserName: string;
    responsibleUserName: string;
    issueStatus: IssueStatus;
    issueType: IssueType;
}

